### PR TITLE
[inference] Remove log about fluid and fix uninitialization warning

### PR DESCRIPTION
**Pull request by zhangjun**
### PR types

Others
### PR changes

Others
### Describe

- Remove log about fluid
- Remove useless forward declarations
- Fix uninitialization warning (trt onehot)

### CODE REVIEW

File: paddle/fluid/inference/tensorrt/convert/silu_op.cc [hunk 1]
```diff
@@ -36,7 +23,7 @@ class SiluOpConverter : public OpConverter {
   void operator()(const framework::proto::OpDesc& op,
                   const framework::Scope& scope,
                   bool test_mode) override {
-    VLOG(4) << "convert fluid silu op to tensorrt layer";
+    VLOG(4) << "convert silu op to tensorrt layer";
```
Comment (jiweibo) [inline paddle/fluid/inference/tensorrt/convert/silu_op.cc:26]: 有的vlog是3，有的vlog是4，这后面需要统一吗？

**Review by jiweibo — APPROVED** [review]
LGTM

File: paddle/fluid/inference/tensorrt/convert/silu_op.cc (hunk 1 above)
Comment (zhangjun) [reply paddle/fluid/inference/tensorrt/convert/silu_op.cc:26]: 后续进行统一

**Review by qingqing01 — APPROVED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**